editor/src/components/editor/server.ts

```diff
@@ -281,12 +281,14 @@ export function assetToSave(fileType: string, base64: string, fileName: string):
   }
 }
 
-export async function saveAssets(projectId: string, assets: Array<AssetToSave>): Promise<void> {
+export async function saveAssets(
+  projectId: string,
+  assets: Array<AssetToSave>,
+): Promise<(string | null)[]> {
```

Comment:
```suggestion
): Promise<Array<string | null>> {
```
This might be slightly clearer.

Reply:
oh that's true! 🙏 https://github.com/concrete-utopia/utopia/pull/2933/commits/e0fa1135b74406a4abc15194f70d06c0f3430b8e